docs/source/configuration_file.md

```diff
@@ -0,0 +1,139 @@
+# Configuration file
+
+* [General parameters](#general-parameters)
+* [Loader parameters](#loader-parameters)
+* [Split dataset](#split-dataset)
+* [Training parameters](#training-parameters)
+* [Architecture](#architecture)
+* [Testing parameters](#testing-parameters)
+* [Transformations](#transformations)
+* [Examples](#examples)
+
+## General parameters
+#### command
+Run the specified command. Choices: ``"train"``, ``"test"``, ``"eval"``, to train, test and evaluate a model respectively.
+#### gpu
+Integer. ID of the GPU to use.
+#### log_directory
+Folder name where output files (e.g. trained model, predictions, results) are saved.
+#### debugging
+Bool. Allows extended verbosity and intermediate outputs.
+
+## Loader parameters
+#### bids_path
+String. Relative path of the BIDS folder.
+#### target_suffix
+Suffix list of the derivative file containing the ground-truth of interest (e.g. [`"_seg-manual"`, `"_lesion-manual"`]). If the list has a length greater than 1, then a multi-class model will be trained.
+#### contrasts
+- `train_validation`: list of image modalities loaded for the training and validation. If `multichannel`, then this list represent the different channels of the input tensors. Otherwise, the modalities are mixed and the model has only one input channel.
+- `test`: list of image modalities loaded in the testing dataset. Same comment than for `train_validation` regarding `multichannel`.
+- `balance`: Dict. Used to under-represent some modalities in the dataset: e.g. `{"T1w": 0.1}` will include only 10% of the available `T1w` images into the training/validation/test set. Please set `multichannel` to `false` if you are using this parameter.
+#### multichannel
+Bool. Indicated if more than a modality is used by the model. See details in both `train_validation` and `test` for the modalities that are input.
+#### slice_axis
+Choice between `"sagittal"`, `"coronal"`, and `"axial"`.
+This parameter decides the slice orientation on which the model will train.
+#### slice_filter
+Dict:
+- `filter_empty_input`: Bool. Filter empty images if `true`.
+- `filter_empty_mask`: Bool. Filter empty GT mask slices if `true`.
+#### roi
+Dict. of parameters about the region of interest
+- `suffix`: suffix of the derivative file containing the ROI used to crop (e.g. `"_seg-manual"`) with `ROICrop` as transform. Please use `null` if you do not want to use a ROI to crop.
+- `slice_filter_roi`: int. It filters ROI mask slices with less than this threshold of non zero voxels. Active when using `"ROICrop"` and inputing ROI file.
```

Comment:
what is meant by "it filters"?  Did you mean the following?
"Voxels contained in the ROI mask and which value are strictly below the value given in this parameter will be replaced by zero.".

Reply:
What about:
"It filters (i.e. discards from the dataset) slices where the ROI mask has with less than this number of non-zero voxels."